openai-java-core/src/main/kotlin/com/openai/core/Utils.kt

```diff
@@ -93,9 +93,11 @@ internal fun Any?.contentToString(): String {
 internal fun isAzureEndpoint(baseUrl: String): Boolean {
     // Azure Endpoint should be in the format of `https://<region>.openai.azure.com`.
     // Or `https://<region>.azure-api.net` for Azure OpenAI Management URL.
+    // Or `<user>-random-<region>.cognitiveservices.azure.com`.
```

Comment:
```suggestion
    // Or `<user>-random-<region>.cognitiveservices.azure.com` for deployments across regions.
```

this is my best guess regarding the reason that I got this URL for this deployment, but I cannot know for sure, if reason is optional I would prefer leaving it empty

Reply:
```suggestion
    // Or `<user>-random-<region>.cognitiveservices.azure.com` for Azure AI services deployments. 
```

It might also be that reason, my openai deployment has the openai.azure.com url, but my azure ai service deployment has cognitiveservices.azure.com

Reply:
I will try to get more details on this, but I think that it's probably okay to not qualify this documentation addition.